ci/e2e/oidc_configuration/cluster_with_managed_oidc_config/main.tf

```diff
@@ -0,0 +1,91 @@
+#
+# Copyright (c) 2023 Red Hat, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+terraform {
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = ">= 4.20.0"
+    }
+    ocm = {
+      version = ">=1.0.1"
```

Comment:
Do we want to keep the version in environment variable and let the ci manage this?
Does this makes sense?

Reply:
I used the example files as template for this tffile. I think it is good idea, to define the version by CI. I would add it in different PR